benches/guest_memory.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2020 Alibaba Cloud Computing. All rights reserved.
+//
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+
+pub use criterion::{black_box, Criterion};
+use vm_memory::{GuestAddress, GuestMemory, GuestMemoryMmap};
+
+const REGION_SIZE: u64 = 0x10_0000;
+const REGIONS_COUNT: u64 = 256;
+
+pub fn benchmark_for_guest_memory(c: &mut Criterion) {
+    benchmark_find_region(c);
+}
+
+fn find_region(mem: &GuestMemoryMmap) {
+    for i in 0..REGIONS_COUNT {
+        let _ = mem.find_region(GuestAddress(i * REGION_SIZE)).unwrap();
+    }
+}
+
+fn benchmark_find_region(c: &mut Criterion) {
+    let mut regions: Vec<(GuestAddress, usize)> = Vec::new();
+    for i in 0..REGIONS_COUNT {
+        regions.push((GuestAddress(i * REGION_SIZE), REGION_SIZE as usize));
+    }
+
+    let memory = GuestMemoryMmap::from_ranges(regions.as_slice()).unwrap();
```

Comment:
nit: this seems to be duplicated in `benchmark_for_mmap()`. It would be nice to have a helper function for it, something like:
`
pub fn create_memory(regions_count: u64, regions_size: size: u64) -> GuestMemoryMmap {}
` 
This way it would be also a bit easier to spot what the benchmark does.

Also, any reason to not keep this file in `benches/mmap` (and eventually add `#![cfg(feature = "backend-mmap")]` to it)?

Reply:
Add a helper to reduce duplicated code.
I aimed to keep consistence between benchmark files and source files:)